test/plugin/test_in_http.rb

```diff
@@ -162,6 +162,29 @@ def test_multi_json_with_time_field
     assert_equal_event_time time, d.events[1][1]
   end
 
+  def test_multi_json_with_time_key
```

Comment:
`test_multi_json_with_time_key` seems to be testing for existent time_key.
I perfer to use `test_multi_json_with_nonexistent_time_key` or `test_multi_json_with_missing_time_key` for test case name.